core/views.py

```diff
@@ -17,12 +17,12 @@ def home():
 
 @app.errorhandler(NotFound)
 def error_404(e):
-    return render_template('error.html', code=404)
+    return render_template('error.html', code=404), 404
```

Comment:
Why do we need both in render_template and in return?

Reply:
It is done this way in [Flask documentation](https://flask.palletsprojects.com/en/1.1.x/quickstart/#rendering-templates). And I think it also allows to return the error code I have added and not only the template.

Reply:
the flask documentation indeed mentions that [it's a parameter of return](https://flask.palletsprojects.com/en/1.1.x/quickstart/#redirects-and-errors).
`code` therefore has no effect in `render_template` and should be removed to avoid confusion.

Reply:
The parameter `code` is used in _error.html_ to display the error code in the view (line 13).